gridpy/network.py

```diff
@@ -72,6 +74,12 @@ def disconnect(self, id: str):
     def dump(self, file: str, format: str = 'XIIDM'):
         _gridpy.dump_network(self.ptr, file, format)
 
+    def reduce(self, min: float = 0, max: float = sys.float_info.max, ids: List[str] = [],
```

Comment:
Maybe we can name `min` and `max` `vmin` and `vmax` instead, just to clarify a little their meaning.

(also a more general comment for the future: we should probably add some "docstrings" at some point if the library gets more used/maintained : it would help understand expected arguments)

Reply:
Yes, docstrings is essential for wide usage of the library. It was on my TODO list but I did had time to start anything.
Also it would be great to have a readthedocs.io with a user doc and part made of generated doc from doncstrings. readthedocs.io seems to be a common way to document Python libraries.